index.html

```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/98db508811.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+
+<body>
+    <div class="background-n">
+
+        <div class="background-opacity">
+            <div class="container">
+                <nav>
+                    <form action="#" method="POST">
+                        <input type="text" name="city" placeholder="Введите название города">
+                        <button> click</button>
+                    </form>
+                    <div class="city-contry">
+                        <p> BELARUS,</p>
+                        <p>MINSK</p>
+                    </div>
```

Comment:
Will be difficult to change this data
You can make this text as 1 element, but add text as 2 separate pieces. Example below:
```js
country = 'Belarus'
city = 'Minsk'

place = `${country},${city}`
```

Reply:
Yep, and you can also create this text as only 1 `<p>` tag. There is no need for 2 separate tags